server/real_estate/src/main/java/com/realestate/real_estate/repos/PropertyDetails.java

```diff
@@ -1,14 +1,23 @@
 package com.realestate.real_estate.repos;
 
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
 import lombok.Builder;
 import lombok.Data;
 import lombok.extern.jackson.Jacksonized;
 
 @Data
 @Builder
 @Jacksonized
+
+@Document(collection = "propertyDetails")
 public class PropertyDetails {
+    @Id
+    int id;
     String description;
     int bedrooms;
     int bathrooms;
+    int price;
+    String district;
```

Comment:
Should we have Strung address as well?

Reply:
Do you think it is a good idea to have a String address?

Reply:
How do you intend to store address of property?

Reply:
What if district can store the address as well? Just a thought.

Reply:
If thats the case let's rename 'district' to 'address'. To avoid future confusion

Reply:
Discussed and agreed